automation/capitanodoc/index.ts

```diff
@@ -17,37 +17,62 @@
 import * as _ from 'lodash';
 import * as path from 'path';
 
-import capitanodoc = require('../../capitanodoc');
+import { getCapitanoDoc } from './capitanodoc';
 import { Category, Document } from './doc-types';
 import * as markdown from './markdown';
 
-const result: Document = {
-	title: capitanodoc.title,
-	introduction: capitanodoc.introduction,
-	categories: [],
-};
-
-for (const commandCategory of capitanodoc.categories) {
-	const category: Category = {
-		title: commandCategory.title,
-		commands: [],
+/**
+ * Generates the markdown document (as a string) for the CLI documentation
+ * page on the web: https://www.balena.io/docs/reference/cli/
+ */
+export async function renderMarkdown(): Promise<string> {
```

Comment:
Most of this code was just refactored inside functions. (The code was bare at the module's global namespace.)